scrapy/http/request/form.py

```diff
@@ -54,10 +55,15 @@ def _urlencode(seq, enc):
     return urlencode(values, doseq=1)
 
 
+def _create_parser_from_response(response, parser_cls):
```

Comment:
Back in the future https://github.com/scrapy/scrapy/pull/471#issuecomment-34008359